recipes/tinygrad/recipe.yaml

```diff
@@ -0,0 +1,137 @@
+context:
+  version: "0.10.1"
+  build_number: 0
+  # see github.com/conda-forge/conda-forge.github.io/issues/1059 for naming discussion
+  torch_proc_type: ${{ "cuda" ~ cuda_compiler_version | version_to_buildstring if cuda_compiler_version != "None" else "cpu" }}
+  tests_to_skip: >
+    _not_a_real_test
+    ${{ "fails `assert Device.DEFAULT in failed_platforms`" if 0 }}
+    ${{ " or (TestLinearizerFailures and test_failure_27)" if linux }}
+    ${{ " or (TestLinearizerFailures and (test_failure_56 or test_failure_57))" if osx }}
+    ${{ "Causes `Fatal Python error: Bus error` + segfault" if 0 }}
+    ${{ " or testCopySHMtoDefault" if linux }}
+    ${{ "timing-sensitive; may fail in our relatively puny CI" if 0 }}
+    ${{ " or test_recursive_pad" if linux }}
+    ${{ "`NotImplementedError: Encountered unknown relocation type 4` for something elf-related" if 0 }}
+    ${{ " or test_backward_sum_acc_dtype" if linux }}
+    ${{ "RuntimeError: Attribute list does not match Module context!" if 0 }}
+    ${{ " or test_bf16_disk_write_read" if linux and cuda_compiler_version != "None" }}
+    ${{ "`error: use of undefined value '@.const.pickledata.7893608704'`" if 0 }}
+    ${{ " or (TestWhisper and (test_transcribe_batch21 or test_transcribe_file1))" if osx }}
+    ${{ "signed int32 overflow" if 0 }}
+    ${{ " or test_float_midcast_int32" if osx }}
+    ${{ "tinygrad.codegen.kernel.KernelOptError: must have tensor cores or TC=2" if 0 }}
+    ${{ " or test_unmerged_ifs" if osx }}
+
+package:
+  name: tinygrad
+  version: ${{ version }}
+
+source:
+  url: https://github.com/tinygrad/tinygrad/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: a22402a8c08da0930720b13a8a66a9c9f99507b15fca9caf31fde63776529ca8
+  patches:
+    - patches/0001-also-install-tinygrad.viz.patch
+    # clang 19 doesn't seem to support -std=metal3.1 yet
+    - patches/0002-downgrade-default-metal-standard.patch
+
+build:
+  number: ${{ build_number }}
+  string: ${{ torch_proc_type }}_py${{ python | version_to_buildstring }}_h${{ hash }}_${{ build_number }}
+  script: python -m pip install . -vv
+  skip:
+    - match(python, "<3.10")
+    # toolchain for 11.8 is not usable in end-user environment
+    - cuda_compiler_version == "11.8"
```

Comment:
Should this use `match()` to make sure you don't run into errors derived of strict string comparison (e.g. 11.8.0 != 11.8, but it should).

Reply:
Good point, thanks